lib/hanami/helpers/form_helper/form_builder.rb

```diff
@@ -363,16 +363,21 @@ def fields_for_collection(name, &block)
         #   <!-- output -->
         #   <label for="delivery-address-city">City</label>
         #   <input type="text" name="delivery[address][city] id="delivery-address-city" value="">
-        def label(content, attributes = {})
+        def label(content, attributes = {}, &blk)
+          if content.is_a?(::Hash)
+            attributes = content
+            content = nil
+          end
```

Comment:
hey, I think you can try to use `**` syntax here:


I mean, instead of this code:
```ruby
        def label(content, attributes = {}, &blk)
           if content.is_a?(::Hash)
             attributes = content
             content = nil
           end
```

you can use this one:
```ruby
        def label(content = nil, **attributes, &blk)
```

WDYT about it?

Reply:
I'll update the PR later today